gcp-native-ts-k8s-ruby-on-rails-postgresql/infra/Pulumi.yaml

```diff
@@ -0,0 +1,21 @@
+name: gcp-rails
+runtime: nodejs
+description: A containerized Ruby on Rails app using managed Kubernetes and PostgreSQL
+template:
```

Comment:
I'm still confused why we put `template` config in some examples (and not the others)... Do you know how they are useful?

Reply:
I think this is only useful when used as a template. I removed this.